lib/cdo/aws/metrics.rb

```diff
@@ -49,6 +49,7 @@ def flush(events)
         )
       rescue => exception
         Honeybadger.notify(exception)
+        puts "Error sending metrics: #{exception.full_message}" if rack_env?(:development)
```

Comment:
🎉 
Just a note that  I learned a dimension value is expected to be a string. The printed exception which would've super helpful is:
```
parameter validator found 4 errors:
  - expected params[:metric_data][1][:dimensions][2][:value] to be a String, got class Integer instead.
  - expected params[:metric_data][4][:dimensions][2][:value] to be a String, got class Integer instead.
  - expected params[:metric_data][7][:dimensions][2][:value] to be a String, got class Integer instead.
  - expected params[:metric_data][10][:dimensions][2][:value] to be a String, got class Integer instead.
 ```